index.bs

```diff
@@ -1898,6 +1956,20 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
         1. If |reportWindows| is an error, return it.
         1. Set |spec|'s [=trigger spec/event-level report windows=] to
             |reportWindows|.
+    1. Let |summaryWindowOperator| be "<code>[=summary window operator/count=]</code>".
```

Comment:
Can we initialize the corresponding field in `|spec|` to this instead of introducing a local variable?